pandas/plotting/_core.py

```diff
@@ -1909,12 +1925,15 @@ def _plot(data, x=None, y=None, subplots=False,
         Place legend on axis subplots
     style : list or dict
         matplotlib line style per column
-    logx : bool, default False
-        Use log scaling on x axis
-    logy : bool, default False
-        Use log scaling on y axis
-    loglog : bool, default False
-        Use log scaling on both x and y axes
+    logx : bool or 'sym', default False
+        Use log scaling or symlog scaling on x axis
+        .. versionadded:: 0.25.0
+    logy : bool or 'sym' default False
```

Comment:
need a blank line after the versionadded (for each of these); make these versionchanged